com.transmissionbt.Transmission.json

```diff
@@ -19,7 +19,8 @@
         "--talk-name=org.freedesktop.Notifications",
         "--filesystem=xdg-run/gvfsd",
         "--talk-name=org.gtk.vfs.*",
-        "--talk-name=org.kde.StatusNotifierWatcher"
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk=name=org.gnome.SessionManager"
```

Comment:
```suggestion
        "--talk-name=org.gnome.SessionManager"
```

Reply:
Used here https://github.com/transmission/transmission/blob/6cee75547cfec3810bf4b649b42acae63af3f538/gtk/Session.cc#L1006

Reply:
Oops. Sorry for the typo :laughing: